configs/components/cpp-pcp-client.rb

```diff
@@ -46,7 +46,10 @@
     pkg.build_requires "pl-cmake"
     pkg.build_requires "pl-boost"
 
-    if platform.is_cisco_wrlinux?
+    # SLES and EL-4 have old versions of gettext on them; until we build our own,
```

Comment:
Updated the comments; I could also restrict to just SLES-10 and EL-4, or even just gettext versions prior to `0.17`. That'd take more effort, and I don't see a reason to get that specific yet.